lib/bindings/langs/swift/Sources/BreezSDKLiquid/SDKNotificationService.swift

```diff
@@ -26,16 +27,20 @@ open class SDKNotificationService: UNNotificationServiceExtension {
             }
             return
         }
-        
+
         if let currentTask = self.getTaskFromNotification() {
             self.currentTask = currentTask
             
             DispatchQueue.main.async { [self] in
                 do {
                     logger.log(tag: TAG, line: "Breez Liquid SDK is not connected, connecting...", level: "INFO")
                     liquidSDK = try BreezSDKLiquidConnector.register(connectRequest: connectRequest, listener: currentTask)
+                    let pluginConfigs = getPluginConfigs()
+                    if let nwcConfig = pluginConfigs?.nwc {
+                        plugins.nwc = try NwcPlugin.register(sdk: liquidSDK!, config: nwcConfig, listener: currentTask)
```

Comment:
I feel like we need a cleaner way to cache the current plugins. It's like we're mixing up how we initialize and cleanup. Maybe we need a plugin manager that takes the pluginConfig on to register the plugins. If the plugins are already registered, return them instead. Have an unregister method to cleanup all plugins. 

Maybe you can repurpose SDKPlugins for this? Wdyt?